### DISCUSSION TITLE
Paper request

### DISCUSSION THREAD

**Discussion by ManuelZ** [jgerstmayr/EXUDYN#68] [General]
Would it be possible to share with me your paper "An Improved Dynamic Model of the Mecanum Wheel for Multibody Simulations"?

**Comment by jgerstmayr** [comment]
just send me a regular email through `reply.exudyn@gmail.com` and I can send it.

**Comment by jgerstmayr** [comment]
It is open access:
https://www.sciencedirect.com/science/article/pii/S0094114X23003191
